src/main/java/edu/cornell/kfs/module/purap/CUPurapConstants.java

```diff
@@ -14,11 +14,11 @@
  */
 public class CUPurapConstants {
     
-    public static final String SPECIAL_HANDLING_NOTE_LINE_1_NAME = "Send Check To:";  //This is how we identify the name in DV's, so to be consistent we're using the same string here

```

Comment:
Excellent question.  Kualico eliminated a lot of these types of comments to be able to pass check style configuration changes.  Since we want to eventually start using Checkstyle more effectively, I thought it makes sense to make this change.  Should I undo this?

Reply:
I think it is fine to keep but in general we try to limit the upgrade changes to changes strictly related to the upgrade. That makes it easier to track what was needed for the actual upgrade.